pkgs/tools/security/proxmark3/proxmark3-rrg.nix

```
@@ -0,0 +1,47 @@
+{ lib, mkDerivation, stdenv, fetchFromGitHub, pkgconfig, gcc-arm-embedded
+, bluez5, qt5, readline
+
+, hardwarePlatform ? "PM3RDV4"
+
+, hardwarePlatformExtras ? ""
+
+, bluetoothSupport ? true
+
+, qtSupport ? true }:
```

```suggestion
, hardwarePlatform ? "PM3RDV4"
, hardwarePlatformExtras ? ""
, bluetoothSupport ? true
, qtSupport ? true }:
```

Is there a use case when we want to disable qt or bluetooth? If not please remove those options.